### PR TITLE
Add back done column to migrations table

### PR DESCRIPTION
**Pull request by jakirkham**
It looks like we lost the done column in the migration table. Attempting to add it back here.

xref: https://github.com/conda-forge/conda-forge.github.io/pull/2090

<hr>

PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Setting the colspan to 8 instead of 7 should fix the issue with rendering the table headers.

**Comment by jakirkham** [comment]
Am now seeing "Done" as expected

Though it looks like something is needed to fix the width of the table title

Screenshot 2024-03-11 at 2 17 38 PM

File: src/components/StatusDashboard/current_migrations.jsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ function TableContent({ collapsed, name, resort, rows, select, sort }) {
     <>
       <thead>
         <tr onClick={select}>
-          <th colSpan={7} className={collapsed ? styles.collapsed : undefined}>
+          <th colSpan={8} className={collapsed ? styles.collapsed : undefined}>
```
Comment (jakirkham) [inline src/components/StatusDashboard/current_migrations.jsx:102]: Thanks Sylvain! 🙏

Is this what we are looking for?

Comment (SylvainCorlay) [reply src/components/StatusDashboard/current_migrations.jsx:102]: I think so.

**Comment by jakirkham** [comment]
Looks like it worked! 🎉

Screenshot 2024-03-11 at 2 25 52 PM

**Review by SylvainCorlay — APPROVED** [review]

**Pull request merged**